ash/src/vk/prelude.rs

```diff
@@ -26,3 +26,8 @@ impl Packed24_8 {
         (self.0 >> 24) as u8
     }
 }
+
+impl super::ColorComponentFlags {
+    /// Contraction of [`Self::R`] | [`Self::G`] | [`Self::B`] | [`Self::A`]
+    pub const RGBA: Self = Self(Self::R.0 | Self::G.0 | Self::B.0 | Self::A.0);
```

Comment:
The unwrap/rewrap dance isn't needed, is it?
```suggestion
    pub const RGBA: Self = Self::R | Self::G | Self::B| Self::A```

Reply:
`BitOr` implemented for `ColorComponentFlags` isn't `const` so I couldn't make that work. See compiler errors in first push, before force-push.